### ISSUE TITLE
Indent with multiline arguments in a chain

### ISSUE THREAD

**Issue by MadLittleMods** [eslint/eslint#4076]
The [`indent`](http://eslint.org/docs/rules/indent) rule should take into account when a chain that starts with multiline function call.

 - eslint v1.6.0

This throws a `Expected indentation` error for each parameter in `geocode` and the closing `})` line; it wants it to be one level less:

```js
function foo() {
    return geocode({
            address: 'some address'
        })
        .then((result) => {
            return result.latlng;
        });
}
```

Currently, this works fine:
```js
function foo() {
    return geocode({ address: 'some address' })
        .then((result) => {
            return result.latlng;
        });
}
```

---

This may be related to https://github.com/eslint/eslint/issues/4075

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by ilyavolodin** [comment]
@gyandeeps @BYK Can you take a look?

**Comment by gyandeeps** [comment]
https://github.com/eslint/eslint/issues/4081#issuecomment-146991265 applies here too.

**Comment by nzakas** [comment]
@gyandeeps can you update the issue labels accordingly?

**Comment by kaicataldo** [comment]
Closing this because we haven't made any progress in months.

**Comment by MadLittleMods** [comment]
@kaicataldo  Why not leave it open until progress happens?

**Comment by ilyavolodin** [comment]
@MadLittleMods It's our policy to close issues that haven't made any progress after 21 days. We had at some point over 300 issues open, and it's really hard to navigate them and fine things you want to work on. So this policy was created to make it easier for people to actually commit code and triage issues.

**Comment by MadLittleMods** [comment]
@ilyavolodin Can I create a new issue for this or how does this get brought up again?

https://tonicdev.com/57bfad8b71dba61400aecf05/57bfad8b71dba61400aecf06/branches/master

**Comment by ilyavolodin** [comment]
@MadLittleMods You can open another issue. We are currently focusing on JSCS compatibility milestone, so there's still a good chance that this issue will go unaddressed for another 21 days.

**Comment by G-Rath** [comment]
@ilyavolodin That sounds like a great way for bugs to get swept under the rug.

**Comment by not-an-aardvark** [comment]
The "21-day policy" applies to enhancement requests, not bugs. The `indent` behavior in this issue is working as intended.
